actionners/kubernetes/drain/drain.go

```diff
@@ -166,14 +181,25 @@ func Action(action *rules.Action, event *events.Event) (utils.LogLine, *model.Da
 					GracePeriodSeconds: gracePeriodSeconds,
 				},
 			}
-			if err := client.PolicyV1().Evictions(pod.GetNamespace()).Evict(context.Background(), eviction); err != nil {
+			if err = client.PolicyV1().Evictions(pod.GetNamespace()).Evict(context.Background(), eviction); err != nil {
 				utils.PrintLog("warning", utils.LogLine{Message: fmt.Sprintf("error evicting pod '%v': %v", p.Name, err)})
 				evictionErrorsCount++
 			}
 		}()
 	}
 
 	wg.Wait()
+	if config.WaitPeriod != 0 {
+		err = verifyEvictionHasFinished(client, config.WaitPeriod, nodeName, config)
```

Comment:
the client can be retrieved in the func directly, no need to pass it as arg

Reply:
I think it's better to receive the client so we can later test this function by passing a mock client, the client would be refactored in a interface and then we'd use that interface as a way to mock it.